package.json

```diff
@@ -219,6 +219,7 @@
     "electron-devtools-installer": "^3.2.0",
     "electron-updater": "^4.6.5",
     "electron-window-state": "^5.0.3",
+    "fetch-mock": "^9.11.0",
```

Comment:
Maybe good for time being, but in the future we should wrap fetch with our own functions, which do one thing and limit the amount of combinations. That will make `fetch-mock` redundant.

Mocking of those functions is enough in behavioural unit tests, the functions themself should have unit tests that makes sure that they work against fetch.

E.g. (details irrelevant)

```typescript
const callJsonApi = async ({ method, url, body }) => {
  const httpResponse = await fetch(url, {
    method,
    body: JSON.stringify(body),
    headers: { "Content-Type": "application/json", Accept: "application/json" },
  });

  return {
    callWasSuccessful: true, // ...whatever it means to get status of the call
    response: await httpResponse.json(),
  };
};
```

This will gives us:
1. Ability to keep interfaces simple and clean
2. Ability to change our mind in future in single place
3. Ability to extend in single place (e.g. add error monitoring for all calls)
4. Ability to stop using try-catch in operational code being wrong tool for the job
5. Ability to override our own implementation in behavioural unit tests.

Reply:
Okay will remove